lsp/src/saros/lsp/SarosLauncher.java

```diff
@@ -0,0 +1,70 @@
+package saros.lsp;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.Socket;
+import java.net.URL;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.eclipse.lsp4j.jsonrpc.Launcher;
+import saros.lsp.extensions.client.ISarosLanguageClient;
+import saros.lsp.log.LanguageClientAppender;
+import saros.lsp.log.LogOutputStream;
+
+/** Entry point for the Saros LSP server. */
+public class SarosLauncher {
+
+  private static final Logger LOG = Logger.getLogger(SarosLauncher.class);
+  private static final String LOGGING_CONFIG_FILE = "/log4j.properties";
+
+  /**
+   * Starts the server.
+   *
+   * @param args command-line arguments
+   * @throws Exception on critical failures
+   */
+  public static void main(String[] args) throws Exception {
```

Comment:
Who starts the server? Is this part of the extension?

Reply:
yes